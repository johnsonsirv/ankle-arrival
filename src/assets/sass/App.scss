.App {
  text-align: center;
  background-color: #fbf7f7;
  height: 662px;
  // height: auto;
  width: 968px;
  margin: 0;
}

a {
  text-decoration: none;
}