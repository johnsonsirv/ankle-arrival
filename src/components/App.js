import React from 'react';
import { Route, Switch, NavLink } from 'react-router-dom';
import { ToastContainer } from 'react-toastify';
import HomePage from './homePage';
// import DoctorDetails from '../containers/doctorDetails';
import AppointmentList from '../containers/appointmentList';
import DoctorList from '../containers/doctorList';
import Wizard from '../wizard';
import Signup from '../auth/signup';
import Login from '../auth/login';
import ProtectedRoute from '../containers/protectedRoute';
import PageNotFound from './notFoundPage';
import NavBar from './navBar';
import 'react-toastify/dist/ReactToastify.css';

function App() {
  return (
    <>
      <ToastContainer />
      <div className="App">
        <NavBar />
        <Switch>
          {/* <ProtectedRoute
            path="/doctors/:id/:username"
            component={DoctorDetails}
          /> */}
          <ProtectedRoute path="/appointments" component={AppointmentList} />
          <ProtectedRoute path="/doctors" component={DoctorList} />
          <Route path="/diagnosis" component={Wizard} />
          <Route path="/signup" component={Signup} />
          <Route path="/login" component={Login} />
          <Route path="/" exact component={HomePage} />
          <Route component={PageNotFound} />
        </Switch>
      </div>
    </>
  );
}

export default App;
