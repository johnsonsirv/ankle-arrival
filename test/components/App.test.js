/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import App from '../src/components/App';

describe('<App /> rendering', () => {
  it('renders correctly', () => {
    let wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should mount in a full DOM', () => {
    expect(shallow(<App />).find('.App')).toHaveLength(1);
  });
  it('renders learn react link', () => {
    let wrapper = shallow(<App />);
    expect(wrapper.find('a').text()).toMatch(/Learn React/);
  });
  it('renders react header selectable by class "App-header"', () => {
    let wrapper = shallow(<App />);
    expect(wrapper.find('header').is('.App-header')).toBe(true);
  });
});
