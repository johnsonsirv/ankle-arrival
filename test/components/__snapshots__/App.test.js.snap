// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> rendering renders correctly 1`] = `ShallowWrapper {}`;
